tests/SideBySide.New/ConnectionPool.cs

```diff
@@ -148,9 +118,5 @@ public void ResetConnection(object connectionReset, int poolSize, long expected)
 				connection.Dispose();
 		}
 
-		public void Dispose()
-		{
-			MySqlConnection.ClearAllPools();
```

Comment:
XUnit instantiates all test classes at the very beginning.  Since pools are shared among tests, `ClearAllPools` wreaks havoc on other tests that are still running.